localization/english/bookmark/bookmark_583_l_english.yml

```
@@ -1,11 +1,40 @@
 ﻿l_english:
  bm_the_invasion_from_draenor:0 "The Invasion from Draenor"
- bm_the_invasion_from_draenor_desc:0 "PH"
+ bm_the_invasion_from_draenor_desc:0 "Years after the end of the Gurubashi war, a new threat of war rises upon Azeroth. The Dark Portal, rift between the worlds of Azeroth and Draenor, has been opened in the Black Marsh east to the unsuspecting kingdom and through it came the Orcish Horde, eager to claim Azeroth for its own.
+ 
+ The Horde's ambitious warchief, Blackhand, is preaparing for a glorious campaign to add a second world to his path of glory, while some other orcs such as the noble chieftain Durotan only wishes to flee their dying world. Whatever might be their motivation, it changes nothing for king Llane and the humans of Stormwind who will fight to the death to defend their homeland.
+ 
+ It is not only a story of kings and warchiefs, though, for unkown to sides, war is a chess game between sorcerers mastermind and their secret agenda. The human Medivh and the orc Gul'Dan opened the Dark Portal for reasons of their own, and they are not done, yet...
+ 
+ #Will the Horde plunge Azeroth into endless bloodshed ? Will the orcs find a new world to inhabit, or will they pushed back to the doomed Draenor ? What darker schemes await for Azeroth ?#!"
 
 
  #Llane Wrynn
  llane_wrynn_character:0 "King Llane"
- llane_wrynn_character_desc:0 "PH"
+ llane_wrynn_character_desc:0 "A knightly king, Llane was raised in the bond of brotherhood with his close friends knight-champion Anduin Lothar and Medivh, the Guardian. Alongside his friends and advisors, the young king defeated the Trolls who killed his father during the Gurubashi War and proved himself true to the legacy of Wrynn dynasty by making his kingdom the mightiest of all humanity. With the invasion from Draenor incoming, Llane now has to face a threat more terrible than any of his ancestors did.
```

- Split some of the sentence so it's not too long.
- Minor fix.
```suggestion
llane_wrynn_character_desc:0 "A knightly king, Llane was raised in the bond of brotherhood with his close friends knight-champion Anduin Lothar and Medivh, the Guardian. Alongside his friends and advisors, the young king defeated the trolls who killed his father during the Gurubashi War. Llane has proved himself true to the legacy of the Wrynn dynasty as Stormwind become the mightiest as ever been. With the invasion from Draenor incoming, Llane now has to face a threat more terrible than any of his ancestors did.
```